dgc/Valuesets/disease-agent-targeted.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "valueSetId": "disease-agent-targeted",
+  "valueSetDate": "2021-04-27",
+  "valueSetValues": {
+    "840539006": {
+      "display": "COVID-19",
+      "lang": "en",
+      "active": true,
+      "version": "http://snomed.info/sct/900000000000207008/version/20210131",
+      "system": "http://snomed.info/sct"
+    }
+  }
+}
```

Comment:
Just a comment: I take it you do know the valuesets should be distributed by the DCCG? If so, then ok with these here as just a snapshot for development purposes only